packages/validation-manager/src/TracerResultParser.ts

```diff
@@ -33,24 +40,12 @@ interface CallEntry {
  * - entries are ordered by the return (so nested call appears before its outer call
  * - last entry is top-level return from "simulateValidation". it as ret and rettype, but no type or address
  * @param tracerResults
+ * @param abi
  */
-function parseCallStack (tracerResults: BundlerCollectorReturn): CallEntry[] {
```

Comment:
this is a local parser. no need to "externalize" the abi . Maybe remove unneeded (test) contracts, and add IAccount instead